ai/commands.py

```diff
@@ -0,0 +1,64 @@
+import os
+import subprocess
+import sys
+
+from .network import Network
+
+
+class Commands:
+    def __init__(self, network: Network, team_name: str):
+        self.network = network
+        self.team_name = team_name
+
+    def send_command(self, command: str) -> str:
+        print(f"Sending: {command}")
+        self.network.send_message(command)
+        response = self.network.receive_message()
+        print(f"Sent: {command}, Received: {response}")
+        return response
+
+    def look(self) -> str:
```

Comment:
This class should be responsible to encapsulate fully each command, meaning it should also transform the commands output to something manageable